### DISCUSSION TITLE
Vote on community based github organization name around vcpkg stuff

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#36427] [Polls]
Follow up on https://github.com/microsoft/vcpkg/discussions/36189. 
Three names have been proposed. Please choose your favorite.

**Deadline for voting: 2024-02-11**

cc @m-kuhn, @autoantwort, @dg0yt, @Osyotr, @cenit, @Thomas1664, @Hoikas, @StarGate-One

**Comment by Thomas1664** [comment]
~~Assuming the 9 votes are from the people you tagged, I think you can close the ballot now. Maybe those people can leave a comment to confirm they already voted?~~

I did vote.

**Reply by m-kuhn** [reply]
I did vote

**Reply by dg0yt** [reply]
Never change the rules while voting!

**Reply by m-kuhn** [reply]
You democrats. Fair point 👍 
I think some are just eager to get started 😄

**Reply by Thomas1664** [reply]
I mean if everyone already voted, the rules aren't changed _while_ voting.

**Reply by Neumann-A** [reply]
> I think some are just eager to get started 

I mean I could already send out invites with the risk that everything has to move to a different org. 

> I mean if everyone already voted, the rules aren't changed while voting.

You simply don't know if everybody voted. I only mentioned people who actively interacted proposing a name but there might be more people who want to decide on the name.

**Reply by Thomas1664** [reply]
> I mean I could already send out invites with the risk that everything has to move to a different org.

Given that there're now more votes than people you mentioned, I withdraw my original request to close the ballot.

> I only mentioned people who actively interacted proposing a name but there might be more people who want to decide on the name.

I remember @strega-nil really liked vcpkg. Maybe she would also be interested in this?

**Reply by m-kuhn** [reply]
> > I think some are just eager to get started
> 
> I mean I could already send out invites with the risk that everything has to move to a different org.

Worst case we'd have to [rename the organization](https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization)

**Reply by StarGate-One** [reply]
I voted. Maybe we need some "I Voted" stickers you get once you vote 👍🏼

**Comment by Neumann-A** [comment]
Results are in:

![grafik](https://github.com/microsoft/vcpkg/assets/30894796/e70aedf2-f9f7-43d7-931e-98a7fd4544c2)

as such the name will be open-vcpkg

**Comment by cenit** [comment]
i just realized one thing: is the vcpkg name freely usable?

**Reply by Thomas1664** [reply]
I'm not a lawyer but I think so. I mean you also have the name "vcpkg" in your fork. And I don't think they registered it as a trademark (at least I couldn't find any indication that they did).

**Reply by Neumann-A** [reply]
As far as I am aware protection names would require the registration of a trademark. vcpkg probably does not full fill the requirements for a trademark anyways but there is no trademark registered as far as the EU and US Databases are concerned.

**Comment by Thomas1664** [comment]
@Neumann-A how do you plan to move forward? Did you already invite people to the organisation?

**Reply by Neumann-A** [reply]
The organization is created under https://github.com/open-vcpkg. I have send an invite out to you

I would like for people to simply apply for membership but that is not how github orgs work. So whoever wants to become a member has to contact one of the admins. Alternativly just ping me here and I'll send out an invite. I just don't wont to send out invites en mass and have them regarded as unwanted spam.

**Reply by Osyotr** [reply]
@Neumann-A 👀

**Reply by cenit** [reply]
@Neumann-A ;)

**Reply by StarGate-One** [reply]
@Neumann-A
